terraform/projects/infra-networking-route53/main.tf

```diff
@@ -57,12 +73,16 @@ data "terraform_remote_state" "app-ecs-albs" {
 }
 
 ## Resources
+# --------------------------------------------------------------
+# These resources are only created for staging or production environments (not dev)
 
 resource "aws_route53_zone" "subdomain" {
-  name = "${var.prometheus_subdomain}.gds-reliability.engineering"
+  count = "${local.create_staging_or_prod_count}"
+  name  = "${var.prometheus_subdomain}.gds-reliability.engineering"
 }
 
-resource "aws_route53_record" "prom-alias" {
+resource "aws_route53_record" "prom_alias" {
```

Comment:
I think this record should be in the app-ecs-albs module; we shouldn't have a circular dependency between `infra-networking` and `app-ecs-albs` because, when creating a new environment, which one do you create first?

Specifically, I imagine that infra-networking creates the zone (with the zone id as an output), and app-ecs-elbs creates the record within the zone.

Reply:
ok, that could work, will then mean that we won't need a separate route53 terraform project